packages/aws-cdk-lib/aws-kms/test/key.test.ts

```diff
@@ -646,6 +646,17 @@ test('fails if key policy has no IAM principals', () => {
   expect(() => app.synth()).toThrow(/A PolicyStatement used in a resource-based policy must specify at least one IAM principal/);
 });
 
+test('creates a multi-Region primary key', () => {
```

Comment:
It's good to keep it simple.

```suggestion
test('multi-region primary key', () => {
```